src/AutorestSwift/View Models/ServiceClientFileViewModel.swift

```diff
@@ -79,7 +79,8 @@ func resolveGlobalParameters(from model: CodeModel) -> ([ParameterViewModel], St
     for globalParameter in model.globalParameters ?? [] {
         if globalParameter.name == "$host" {
             host = globalParameter.value.clientDefaultValue
-        } else {
+            // `endpoint` and `api-version` have different logic in generated code. Do not add them to the list of global parameters
+        } else if !["endpoint", "api-version"].contains(globalParameter.serializedName) {
```

Comment:
Should we add `apiVersion` to this list as well?

Reply:
I did not because the logic we have for getting api version is looking for 'api-version' only. See this in CodeModel.swift. If we add `apiVersion`, we need to do in both places.

```
    func getApiVersion() -> String {
        if let constantSchema = globalParameter(for: "api-version")?.schema as? ConstantSchema {
            return constantSchema.value.value
        } else {
            return ""
        }
    }
```

Reply:
So is it expected for autorest swift to support both apiVersion and api-version in swagger?

Reply:
Nope, this is good.